index.test.js

```diff
@@ -16,6 +16,8 @@ const mockEcsUpdateService = jest.fn();
 const mockEcsDescribeServices = jest.fn();
 const mockCodeDeployCreateDeployment = jest.fn();
 const mockCodeDeployGetDeploymentGroup = jest.fn();
+const mockRunTask = jest.fn();
```

Comment:
Add a `mockRunTaskFailure` with implementations to add the following failure path unit tests (not an exclusive list)

1. A failing RunTask with `wait-for-task-stopped: true`
2. A failing RunTask with Service to make sure Service is not ran on a failure.

Optional but recommended:
3. A failing RunTask with `wait-for-task-stopped: false`